website/templates/project/registrations.mako

```diff
@@ -22,23 +30,147 @@
     ##      %else:
     ##          There have been no registrations of the parent project (<a href="${parent_node['url']}">${parent_node['title']}</a>).
     ##      %endif
-    % else:
-        <div class="m-md">
-            There have been no registrations of this ${node['node_type']}.
-            For a list of the most viewed and most recent public registrations on the
-            Open Science Framework, click <a href="/explore/activity/#newPublicRegistrations">here</a>.
-        </div>
-    % endif
-    %if parent_node['exists'] and user['is_admin_parent']:
-        <div class="m-l-md">To register the entire project "${parent_node['title']}" instead, click <a href="${parent_node['registrations_url']}">here.</a></div>
-    %endif 
-
-  </div>
-  <div class="col-sm-3">
-    <div class="m-md">
-        % if 'admin' in user['permissions'] and not disk_saving_mode:
-          <a id="registerNode" href="${node['url']}register" class="btn btn-success" type="button">New Registration</a>
+        % else:
+        <p>
+          There have been no completed registrations of this ${node['node_type']}.
+          For a list of the most viewed and most recent public registrations on the
+          Open Science Framework, click <a href="/explore/activity/#newPublicRegistrations">here</a>,
+          or you start a new draft registration from the "Draft Registrations" tab.
+        </p>
         % endif
+        %if parent_node['exists'] and user['is_admin_parent']:
+        <br />
+        <br />
+        To register the entire project "${parent_node['title']}" instead, click <a href="${parent_node['registrations_url']}">here.</a>
+        %endif         
+      </div>
     </div>
   </div>
+  <div role="tabpanel" class="tab-pane" id="drafts">
+    <div id="draftRegistrationScope" class="row" style="min-height: 150px">
+      <div data-bind="visible: !preview()">
+        <div class="col-md-9">
+          <h2> Draft Registrations </h2>
+        </div>
+        <div class="col-md-9">
+          <div>
+            % if 'admin' in user['permissions'] and not disk_saving_mode:
```

Comment:
If the user is not admin, and there are no pending draft registrations, they get a header and a blank page. It might help to include some explanatory text on what a draft registration is. (and why a user with Read-only access can't do anything)

![screen shot 2015-09-14 at 10 43 23 am](https://cloud.githubusercontent.com/assets/2957073/9852756/f94c86b0-5acd-11e5-9d7f-3eabbfe1b0a0.png)